ag_data/simulator.py

```diff
@@ -193,6 +193,20 @@ def logSingleMeasurement(self, timestamp):
                 sensor_id=self.sensor,
                 value={"internal": gauss(15, 3), "external": gauss(20, 2)},
             )
+        elif self.checkSensorFormat(2):
+            import math
+
+            second = timestamp.second
+
+            x = math.cos(second / (2 * math.pi))
+            y = math.sin(second / (2 * math.pi))
```

Comment:
The simulation creates oscillation around the unit circle.

Reply:
Thanks Jack.